### PR TITLE
Go SDN packages

### PR DESCRIPTION
**Pull request by 3d0c**
Please check if what you want to add to `awesome-go` list meets [quality standards](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#quality-standard) before sending pull request. Thanks!

**Please provide package links to:**

- github.com repo:
- godoc.org:
- goreportcard.com:
- coverage service link ([cover.run](https://cover.run/), [gocover](http://gocover.io/), [coveralls](https://coveralls.io/) etc.), example: `cover.run go`

Very good coverage

**Note**: that new categories can be added only when there are 3 packages or more.

**Make sure that you've checked the boxes below before you submit PR:**
- [x] I have added my package in alphabetical order
- [x] I know that this package was not listed before
- [x] I have added godoc link to the repo and to my pull request
- [x] I have added coverage service link to the repo and to my pull request
- [x] I have added goreportcard link to the repo and to my pull request
- [x] I have read [Contribution guidelines](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#contribution-guidelines) and [Quality standard](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#quality-standard).

Thanks for your PR, you're awesome! :+1:

### CODE REVIEW

**Comment by kirillDanshin** [comment]
seems like a bug in our tests, again

**Comment by kirillDanshin** [comment]
anyway, we have one rule: new categories can be created only when we have 3 or more links for it.

**Comment by 3d0c** [comment]
More packages added

File: README.md [hunk 1]
```diff
@@ -808,6 +809,15 @@ See [go-hardware](https://github.com/rakyll/go-hardware) for a comprehensive lis
 * [winrm](https://github.com/masterzen/winrm) - A Go WinRM client to remotely execute commands on Windows machines
 * [xtcp](https://github.com/xfxdev/xtcp) - A TCP Server Framework with simultaneous full duplex communication,graceful shutdown,custom protocol.
 
+## Network Virtualization
+
+*Tools and libraries for Virtual Networking and SDN (Software Defined Networking).*
+
+* [Gofc] (https://github.com/Kmotiko/gofc) Openflow 1.3 controller
+* [libovsdb] Open vSwitch Database Protocol implementation
```
Comment (kirillDanshin) [inline README.md]: no link here

File: README.md [hunk 2]
```diff
@@ -808,6 +809,15 @@ See [go-hardware](https://github.com/rakyll/go-hardware) for a comprehensive lis
 * [winrm](https://github.com/masterzen/winrm) - A Go WinRM client to remotely execute commands on Windows machines
 * [xtcp](https://github.com/xfxdev/xtcp) - A TCP Server Framework with simultaneous full duplex communication,graceful shutdown,custom protocol.
 
+## Network Virtualization
+
+*Tools and libraries for Virtual Networking and SDN (Software Defined Networking).*
+
+* [Gofc] (https://github.com/Kmotiko/gofc) Openflow 1.3 controller
```
Comment (kirillDanshin) [inline README.md]: spaces between [] and () will break gh md parser

**Comment by avelino** [comment]
new rules 8 days with label `pending-submitter-response`: https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#maintainers

**Pull request closed without merging**